src/chatClient.cpp

```diff
@@ -3132,6 +3132,11 @@ void GroupChatRoom::initChatTitle(const std::string &title, int isTitleEncrypted
     });
 }
 
+bool GroupChatRoom::isChatdChatInitialized()
```

Comment:
This method would be more appropriate in `ChatRoom`, despite only groupchats need it by now, but could be usefull for 1on1 in a future.